net/chrony/files/chrony.config

```diff
@@ -1,19 +1,35 @@
 config pool
 	option hostname '2.openwrt.pool.ntp.org'
 	option maxpoll '12'
-	option iburst 'yes'
+	option maxsources '4'
+	option iburst '1'
+	option prefer '1'
+	# option xleave '0'
 
-config dhcp_ntp_server
-	option iburst 'yes'
-	option disabled 'no'
+config dhcp_ntp_server 'dhcp_ntp_server'
+	option iburst '1'
+	option disabled '0'
 
-config allow
+config allow 'allow'
 	option interface 'lan'
 
-config makestep
+config makestep 'makestep'
 	option threshold '1.0'
 	option limit '3'
 
-config nts
-	option rtccheck 'yes'
-	option systemcerts 'yes'
+config nts 'nts'
+	option rtccheck '1'
+	option systemcerts '1'
+
+#config smoothtime 'smoothtime'
+	# option maxppm '400'
+	# option maxwander '0.01'
+	# option leaponly '0'
+
+#config systemclock 'systemclock'
+	# option precision '8e-6'
+	# option leapsecmode 'slew'
+	# option leapsectz 'right/UTC'
```

Comment:
I think the leapsectz directive won't work correctly on musl, only glibc, so not sure if it makes sense to support it. leapsecfile would be better, assuming it can be provided by the user.

Reply:
The [TZDB](https://github.com/openwrt/luci/blob/master/modules/luci-base/ucode/zoneinfo.uc) is updated from time to time by [this script](https://github.com/openwrt/luci/blob/master/build/zoneinfo2ucode.pl) - so there are basically no files. Is extending the TZ info an idea?

Reply:
I'd leave it out until the option actually does something useful on a stock openwrt system. I doubt the openwrt maintainers would be interested in packaging the leapfile or right/UTC timezone (and adding the leap-second conversion support to musl) for such an obscure use case. Advanced users can specify it in /etc/chrony/chrony.conf.

Reply:
OK. Done.